docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+* Sign in.
+* Once you are signed in go [here](here: https://api.portal.trustedservices.intel.com/EPID-attestation)
+* Subscribe for development unlikable.
+* You will receive a SPID, Primary Key and Secondary Key.
+* Use these information to replace the appropriate places in the commands below.
+
+```bash
+echo '[YOUR_SPID]' > ${FPC_PATH}/config/ias/spid.txt
+echo '[YOUR_PRIMARY_KEY]' > ${FPC_PATH}/config/ias/api_key.txt
+echo 'epid-unlinkable' > ${FPC_PATH}/config/ias/spid_type.txt
+```
+
+### Known Issues
+If you don't have an account and directly try to *Subscribe* to the EPID-attestation
+[here](https://api.portal.trustedservices.intel.com/EPID-attestation). ![Subscribe to
+EPID-attestation api](pictures/SubscribeButton.png) And you are not signed in already with the account
+trough [this](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html) then you
+will be lead to the page. ![broken account creation link](pictures/CreateIDZAccount.png)
+If you then click on *create new IDZ account* you will land on a
+[page](https://software.intel.com/registration/?lang=en-us) saying that the site is held hostage.
+Hence, you won't be able to register an account here. So please us the site
+[here](https://www.intel.com/content/www/us/en/forms/basic-intel-registration.html) to register
+yourselfe and just sign in on the top right on the [api
+page](https://api.portal.trustedservices.intel.com/EPID-attestation) with the previously created
+credentials.
+
+## Building the FPC dev container
+Edit the config.override.mk to set HW mode and DOCKER_BUILDKIT=1
+```bash
+vim $FPC_PATH/config.override.mk
+```
+paste in the following
+```bash
+export SGX_MODE=HW
+export DOCKER_BUILDKIT=1
+```
+Now we can build the container
+```bash
+cd ${FPC_PATH}/utils/docker
+make pull-dev
+make run-dev
+#after make run-dev you will end up in a shell in the dev container, you need to exit out
+#otherwise the container stops if you ever accidentally exit the container 
+exit
+```
+## Final preparation of the dev-container
+Fist start the container and then open a shell. **Important** don't forget to exit out of the
+container. The make run-dev commands leaves you in a shell of the fpc-development-main container.
+However, if you close that terminal windows the container stops. So exit out, start it again,
+because that the fpc-development-main container will keep running even though you close your
+terminal.
+```bash
+docker start fpc-development-main
+#you will need that command later to open multiple shells if you want
+docker exec -i -t fpc-development-main bash
+```
+**Disclaimer** the following *make* step also executes integration tests. However, these sometimes fail with.
+```
+got unexpected status: SERVICE_UNAVAILABLE -- no Raft leader
+
+```
+If this keeps happening to you, than you can comment out the failing tests in the Makefile
+```
+vim $FPC_PATH/integration/Makefile
+``` 
```

Comment:
mh, I am hesitating a bit with this section. It' is not really necessary for the deployment on azure I would say. 

This addresses a multi-shell setup, which is desired to have, in particular, in the test-network tutorial. I personally use tmux inside the FPC dev container. However, using tmux on the host and running multiple panes with a bash session inside the dev container certainly makes more sense for this deployment scenario.

Reply:
This section could also move to the main readme in some form. Maybe a subsection "Working with the FPC dev container"?! WDYT?

Reply:
However, I want to leave the "raft unavailable hint in there" because i got stuck on that quiet some time before I asked.